websites/N/NowPlaying/dist/metadata.json

```diff
@@ -0,0 +1 @@
+{ "$schema": "https://schemas.premid.app/metadata/1.0", "author": { "name": "STEPHAN", "id": "183233556150091776" }, "url": [ "nowplaying.stephantollenaar.nl", "playing.stephantollenaar.nl", "spotify.stephantollenaar.nl" ], "description": { "en": "This widget will show your current music track live on your stream! The all in one now playing widget solution for OBS and other live streaming software.", "nl": "Deze widget laat de muziek die je afspeelt direct zien op jouw livestream! De alles in één now playing oplossing voor OBS en andere live streaming software." }, "service": "NowPlaying", "version": "1.0.0", "logo": "https://i.imgur.com/3vR3TUS.png", "thumbnail": "https://i.imgur.com/FMqOw6p.png", "color": "#36414d", "tags": [ "obs", "twitch", "livestreaming", "streaming", "video" ], "category": "socials" }
```

Comment:
bro stephan kan je ook eens op enter drukken?

Reply:
Dat kun je denk ik beter vragen aan die website in jullie documentatie waarmee je de metadata file kan genereren.

Reply:
@eggsywashere

Reply:
Does your website generate it like that?

Reply:
It downloads* that way. Don't really know a way to download a file with formatted code ._.

Reply:
oof

Reply:
format it [here](https://jsonformatter.curiousconcept.com/) and push the changes

Reply:
format it [here](https://jsonformatter.curiousconcept.com/) and push the changes